integration/create_test.go

```diff
@@ -26,6 +27,57 @@ import (
 	"github.com/FerretDB/FerretDB/integration/setup"
 )
 
+func TestCreateStressPostgres(t *testing.T) {
+	setup.SkipForPostgresWithReason(t, "https://github.com/FerretDB/FerretDB/issues/1206")
+	setup.SkipForTigrisWithReason(t, "Tigris needs a schema for collection creation")
+
+	t.Parallel()
+
+	ctx, collection := setup.Setup(t) // no providers there, we will create collections concurrently
+	db := collection.Database()
+
+	collNum := 10
+
+	var wg sync.WaitGroup
+	for i := 0; i < collNum; i++ {
+		wg.Add(1)
+
+		go func(i int) {
+			collName := fmt.Sprintf("stress_%d", i)
```

Comment:
I think we need an extra trigger to start (not just create) all goroutines at the same time. Something like:

```go
ready := make(chan struct{}, collNum)
start := make(chan struct{})

// ...
go func(i int) {
  ready <- struct{}
  <-start
  // ...
}

// ...

for i := 0; i < collNum; i++ {
  <- ready
}

close(start)
```

Reply:
I understand the idea with `start`, but what is `ready` for? Is it to replace `wg`?

Reply:
To start all created goroutines when they are ready, not just when they are created.

We could use a wait group for that, but a different variable.